RecoPixelVertexing/PixelTriplets/src/CAHitQuadrupletGenerator.cc

```diff
@@ -242,12 +243,12 @@ void CAHitQuadrupletGenerator::hitNtuplets(const IntermediateHitDoublets& region
       if (useBendingCorrection) {
         // Following PixelFitterByConformalMappingAndLine
         const float simpleCot = (gps.back().z() - gps.front().z()) / (gps.back().perp() - gps.front().perp());
-        const float pt = 1.f / PixelRecoUtilities::inversePt(abscurv, es);
+        const float pt = 1.f / PixelRecoUtilities::inversePt(abscurv, *theField);
         for (int i = 0; i < 4; ++i) {
           const GlobalPoint& point = gps[i];
           const GlobalError& error = ges[i];
           bc_r[i] = sqrt(sqr(point.x() - region.origin().x()) + sqr(point.y() - region.origin().y()));
-          bc_r[i] += pixelrecoutilities::LongitudinalBendingCorrection(pt, es)(bc_r[i]);
+          bc_r[i] += pixelrecoutilities::LongitudinalBendingCorrection(pt, *theField)(bc_r[i]);
```

Comment:
@slava77 
> Please clarify if the investigation was completed.

Not quite (I lost yesterday because of CERN's LDAP issues). I have found (in 140.56) that this specific call (in `CAHitTripletGenerator` as well) somehow makes the difference. That happens even if in `PixelRecoUtilities::FieldAt0::FieldAt0()` I set the `FieldAt0::fieldInInvGev` from the `MagneticField::inverseNominalValueInGeV()` instead of calculating it in the `FieldAt0` constructor.